File: model/src/main/java/com/zj/model/indices/WeatherLifeIndicesBean.java
package com.zj.model.indices;

import com.zj.model.Refer;

import java.util.List;

public class WeatherLifeIndicesBean {
	private String fxLink;
	private String code;
	private Refer refer;
	private List<WeatherLifeIndicesItem> daily;
	private String updateTime;

	public String getFxLink(){
		return fxLink;
	}

	public String getCode(){
		return code;
	}

	public Refer getRefer(){
		return refer;
	}

	public List<WeatherLifeIndicesItem> getDaily(){
		return daily;
	}

	public String getUpdateTime(){
		return updateTime;
	}

	public static class WeatherLifeIndicesItem{
		private String date;
		private String level;
		private String name;
		private String text;
		private String type;
		private String category;

		public String getDate(){
			return date;
		}

		public String getLevel(){
			return level;
		}

		public String getName(){
			return name;
		}

		public String getText(){
			return text;
		}

		public String getType(){
			return type;
		}

		public String getCategory(){
			return category;
		}
	}


}

File: src/main/kotlin/model/indices/WeatherLifeIndicesBean.kt
package model.indices

import model.Refer
import utils.lifePrefix

data class WeatherLifeIndicesBean(
    val fxLink: String? = null,
    val code: String? = null,
    val refer: Refer? = null,
    val daily: List<WeatherLifeIndicesItem>? = null,
    val updateTime: String? = null,
) {

    data class WeatherLifeIndicesItem(
        val date: String? = null,
        val level: String? = null,
        val name: String? = null,
        val text: String? = null,
        val type: String? = null,
        val category: String? = null,
        var imgRes: String = "${lifePrefix}ic_life_sport.svg",
    )
}